src/sql/helpers.js

```diff
@@ -1,4 +1,4 @@
-const jsonbObjectAgg = (aggregationColumnKey, nestedFields, aggregationJsonKey, sql) => {
+const createCollapsedObject = (collapseKey, nestedFields, collapseKeyNewName, sql) => {
```

Comment:
rename collapseKeyNewName to collapsedKeyNewName to match the name in collapseKeyIntoArray function